sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -2203,8 +2201,9 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
               ContentLength; // Required for "Append Data" and "Flush Data".  Must be 0 for "Flush
                              // Data".  Must be the length of the request content in bytes for
                              // "Append Data".
-          Azure::Core::Nullable<std::string> ContentMd5; // Specify the transactional md5 for the
-                                                         // body, to be validated by the service.
+          Azure::Core::Nullable<Storage::ContentHash>
+              ContentMd5; // Specify the transactional md5 for the body, to be validated by the
```

Comment:
Why not `TransactionalContentHash`?

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1237.
Opened an issue to track this, the doc is wrong, not the type itself.

Reply:
The variable name is also wrong

Reply:
The variable name is not visible to customers.